README.md

```diff
@@ -5,3 +5,5 @@
 This is an opinionated library to share code and approach to Datadog logging in prima.it
 
 Refer to the [official docs](https://docs.rs/prima_datadog) for help on how to setup the library in your project
+
+❕ Please note that `prima_datadog.rs` uses [DogstatsD](https://docs.datadoghq.com/developers/dogstatsd/), which means metrics will be sent using the **UDP** protocol on port `8125`. You can find more information on the official Datadog documentation
```

Comment:
Petty nitpick incoming:
The port needs to be specified by the user, 8125 is what the datadog agent listens on by default.

Reply:
Accepted your suggestion, it felt a bit vague but at the same time the official docs are where it's at

Reply:
I meant that if the user doesn't specify a port(eg. `100.100.100.100`) we won't default to 8125, the user needs to specify which port they want(eg. by doing `100.100.100.100:8125`)